src/main/java/xyz/groundx/caver_ext_kas/kas/kip7/KIP7.java

```diff
@@ -150,19 +159,191 @@ public Call deployAsync(String name, String symbol, int decimals, BigInteger ini
+     *
+     * }
+     * </pre>
+     *
+     * @param name The name of KIP-7 token.
+     * @param symbol The symbol of KIP-7 token.
+     * @param decimals The number of decimal places the token uses.
+     * @param initialSupply The total amount(hex string) of token to be supplied initially.
+     * @param alias The alias of KIP-7 token. Your `alias` must only contain lowercase alphabets, numbers and hyphens and begin with an alphabet.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call deployAsync(String name, String symbol, int decimals, String initialSupply, String alias, ApiCallback<Kip7DeployResponse> callback) throws ApiException {
+        return deployAsync(name, symbol, decimals, initialSupply, alias, null, callback);
+    }
+
+    /**
+     * Deploy KIP-7 token contract with a Klaytn account in KAS.<br>
+     * If you want to see more detail about configuring fee payer option, see <a href="https://refs.klaytnapi.com/en/kip37/latest#section/Fee-Payer-Options">Fee payer options</a><br>
+     * POST /v1/contract
+     *
+     * <pre>Example :
+     * {@code
+     * String testAlias = "test-contract";
+     * String name = "TEST_KIP7";
+     * String symbol = "TKIP7";
+     * int decimals = 18;
+     * BigInteger initialSupply = BigInteger.valueOf(100_000).multiply(BigInteger.TEN.pow(18)); // 100000 * 10^18
+     *
+     * // Using a user fee payer options.
+     * // It needs to have userFeePayer account and KRN created by KAS Wallet API.
+     * String feePayer = "0x{feePayer}";
+     * String feePayer_krn = "krn";
+     *
+     * Kip7FeePayerOptionResponseUserFeePayer userFeePayerOption = new Kip7FeePayerOptionResponseUserFeePayer();
+     * userFeePayerOption.setAddress(userFeePayer.getAddress());
+     * userFeePayerOption.setKrn(userFeePayer.getKrn());
+     *
+     * Kip7FeePayerOption option = new Kip7FeePayerOption();
+     * option.setEnableGlobalFeePayer(false);
+     * option.setUserFeePayer(userFeePayerOption);
+     *
+     * Kip7TransactionStatusResponse response = caver.kas.kip7.deploy(name, symbol, decimals, initialSupply, testAlias, option);
+     * }
+     * </pre>
+     *
+     * @param name The name of KIP-7 token.
+     * @param symbol The symbol of KIP-7 token.
+     * @param decimals The number of decimal places the token uses.
+     * @param initialSupply The total amount of token to be supplied initially.
+     * @param alias The alias of KIP-7 token. Your `alias` must only contain lowercase alphabets, numbers and hyphens and begin with an alphabet.
+     * @param option The feePayer options that config to pay transaction fee logic.
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7DeployResponse deploy(String name, String symbol, int decimals, BigInteger initialSupply, String alias, Kip7FeePayerOption option) throws ApiException {
```

Comment:
How about changing name to `Kip7FeePayerOptions`?
We need to modify yaml file name and title for this.

Reply:
I modified Kip7FeePayerOption to Kip7FeePayerOptions